### ISSUE TITLE
Include CSS in generated HTML file instead of referencing

### ISSUE THREAD

**Issue by ControlTheBit** [neilsustc/vscode-markdown#57]
If I print the current markdown into an HTML file and send it to one of my colleges, that do not have VS Code installed, the HTML looks like crap because the CSS files in there are references to files on my local machine.

```
        <link rel="stylesheet" type="text/css" href="file:///c%3A/Users/butzl/.vscode/extensions/yzhang.markdown-all-in-one-0.11.1/media/markdown.css">
        <link rel="stylesheet" type="text/css" href="file:///c%3A/Users/butzl/.vscode/extensions/yzhang.markdown-all-in-one-0.11.1/media/tomorrow.css">
        <link rel="stylesheet" type="text/css" href="file:///c%3A/Users/butzl/.vscode/extensions/yzhang.markdown-all-in-one-0.11.1/media/checkbox.css">

```

I think it would be great if the information from the file would be included into the HTML file. If this would be implemented, the HTML would also be independent from the used plugin version in VS Code.

**Comment by neilsustc** [comment]
I agree with you. In fact, currently the feature of exporting to HTML file is the first step of exporting to PDF.

You can manually open the exported HTML in your browser, print it to PDF and then share with others *before I get time to make it better*

**Comment by neilsustc** [comment]
Fixed in v0.11.2
